pkg/controllers/user/pipeline/remote/github/github.go

```diff
@@ -104,8 +104,16 @@ func (c *client) CreateHook(pipeline *v3.Pipeline, accessToken string) (string,
 	if err != nil {
 		return "", err
 	}
+	events := []string{}
+	if pipeline.Spec.TriggerWebhookPush {
+		events = append(events, "push")
+	}
+	if pipeline.Spec.TriggerWebhookPr {
```

Comment:
should it be elseif? so the condition doesn't have to be evaluated twice when the first if returns ok

Reply:
it can be both, so that either events can trigger a build